tests/acceptance/request_test.py

```diff
@@ -99,6 +114,7 @@ def test_400_if_required_body_is_missing(test_app):
     ).status_code == 400
 
 
+@pytest.mark.xfail
```

Comment:
I think we'll want to preserve the existing behaviour of this test, so defaulting to `json` when it's missing.